docs/source/ecf.rst

```diff
@@ -30,9 +30,16 @@ Algorithm to use to fit a ramp to the frame-level images of uncalibrated files.
 
 
 maximum_cores
-''''''''''''''''''
+'''''''''''''
 Fraction of processor cores to use when computing the jump step and the ramp fits. Options are ``''none'``, ``'quarter'``, ``'half'``, or ``'all'``.
 
+jump_rejection_threshold
+''''''''''''''''''''''''
+A floating-point value that sets the sigma threshold for jump detection. The default is 4.0, but it is often best to increase this number for time-series observations to avoid excessively high false-positives. The optimal value will vary between different datasets and different instruments, but from experience we have found that values around 6.0--8.0 are often reasonable.
+
+minimum_sigclip_groups
+''''''''''''''''''''''
+The minimum number of groups to switch the jump detection to use sigma clipping. The default is 100.
```

Comment:
Honestly, I'm not 100% sure that this parameter is really relevant for TSO data. One of my colleagues was saying that a spurious step function was occurring between the second-last and last segments because the final segment had a small number of integrations, and they said that increasing this value to 120 resolved their issue. I don't see how this parameter could be relevant in that case though since the number of groups is fixed while the number of integrations does vary between segments. My strong suspicion is that the issue was actually just resolved by a `jwst` pipeline version upgrade and/or new CRDS pmap. As a result, it's not clear to me if/when this value should ever be changed

Reply:
@kevin218, is there anything that you still want me to do here?

Reply:
Nope, nothing else.